packages/tools/src/types/SplineProps.ts

```diff
@@ -10,6 +10,9 @@ export type SplineProps = {
    */
   resolution?: number;
 
+  /** Fixed resolution (Linear spline) */
```

Comment:
requires more info, what can the values be? 0? 1? what do they represent

Reply:
I added a few more info but what you are looking for is already documented on `resolution` property.